app/controllers/tag_controller.rb

```diff
@@ -17,7 +17,7 @@ def index
       .select('node.nid, node.status, term_data.*, community_tags.*')
       .where('node.status = ?', 1)
       .where('community_tags.date > ?', (DateTime.now - 1.month).to_i)
-      .where("name LIKE :prefix", prefix: "#{prefix}%")
+      .where("name LIKE :prefix", prefix: "%#{prefix}%")
```

Comment:
The name **prefix** here is a bit misleading IMO

Reply:
Keyword, word, query, or even just q.

Reply:
updated !